com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -933,6 +933,14 @@ private void downloadSymbols(IProgress progress) throws IOException, CompileExce
                         // We default to the default architectures for the platform, and take the option value
                         // only if it has been set.
                         Platform platform = this.getPlatform();
+                        if (platform != Platform.getHostPlatform() && ExtenderUtil.hasExtensionPipelinePlugins(this)) {
+                            logInfo("Building plugins for host platform");
+
+                            String[] architectures = Platform.getHostPlatform().getArchitectures().getDefaultArchitectures();
+                            buildEngine(monitor, architectures, appmanifestOptions);
+
+                        }
```

Comment:
We need to build the plugins before we build the content.
We only build the tools for the host platform.